diff '--color=auto' -Naur --exclude .git a/GNUmakefile b/GNUmakefile
--- a/GNUmakefile	2023-02-04 08:52:06.332428109 +0800
+++ b/GNUmakefile	2023-02-04 09:16:37.176322899 +0800
@@ -24,7 +24,8 @@
 
 config.h:
 	[ -d blocks ] || cp -R blocks.def blocks
-	sed '2s=<path to the folder containing block scripts>=${EFBLOCKS}=' config.def.h >$@
+	cat config.def.h > $@
+	# sed '2s=<path to the folder containing block scripts>=${EFBLOCKS}=' config.def.h >$@
 
 sigdwmblocks/sigdwmblocks: sigdwmblocks/sigdwmblocks.c
