diff '--color=auto' -Naur --exclude .git a/Makefile b/Makefile
--- a/Makefile	2023-02-04 12:19:23.076664683 +0800
+++ b/Makefile	2023-02-04 12:20:56.172795537 +0800
@@ -43,14 +43,11 @@
 	mkdir -p ${DESTDIR}${MANPREFIX}/man1
 	sed "s/VERSION/${VERSION}/g" < dwm.1 > ${DESTDIR}${MANPREFIX}/man1/dwm.1
 	chmod 644 ${DESTDIR}${MANPREFIX}/man1/dwm.1
-	mkdir -p /usr/share/xsessions
-	cp -f dwm.desktop /usr/share/xsessions/dwm.desktop
 	cp -f scripts/* ${EFSCRIPTS}/
 	chmod +x ${EFSCRIPTS}/*
 
 uninstall:
 	rm -f ${DESTDIR}${PREFIX}/bin/dwm\
-		${DESTDIR}${MANPREFIX}/man1/dwm.1 \
-		/usr/share/xsessions/dwm.desktop
+		${DESTDIR}${MANPREFIX}/man1/dwm.1
 
 .PHONY: all options clean dist install uninstall
